cmd/frontend/graphqlbackend/org_invitations.go

```diff
@@ -562,6 +569,17 @@ func sendOrgInvitationNotification(ctx context.Context, db database.DB, org *typ
 	})
 }
 
+func subjectMatchesOrgId(claims *orgInvitationClaims, invite *database.OrgInvitation) error {
+	orgID, err := strconv.ParseInt(claims.Subject, 10, 32)
+	if err != nil {
+		return err
+	}
+	if orgID != int64(invite.OrgID) {
+		return errors.Newf("token subject %v does match orgId %v", orgID, claims.InvitationID)
```

Comment:
should we use `%d` notation here, as we always show integers?
```suggestion
		return errors.Newf("token subject %d does match orgId %d", orgID, claims.InvitationID)
```